### ISSUE TITLE
Nav tag with `from` parameter always returns empty

### ISSUE THREAD

**Issue by mikemartin** [statamic/cms#1683]
## Bug Description

There seems to be an issue with the `from` parameter on the nav tag. When I add `from="/"` or `from="/{segment_1}"` the structure collection always returns empty. 

```
{{ nav from="/" }}
        <a href="{{ url }}" >{{ title }}</a>
{{ /nav }}
```

## Environment

**Statamic version:** 3.0.0.-beta-26

**PHP version:** 7.{?}

**Install method (choose one):**
- Fresh install from `statamic/statamic`
- Starter kit
- Existing Laravel app
- Other (please explain this magic!)

**Comment by Pfeifakopf** [comment]
Could be the same:
https://github.com/statamic/cms/issues/1542

**Comment by jasonvarga** [comment]
Do you have a collection named pages? What's in its yaml file?

**Comment by edalzell** [comment]
```
title: Pages
template: subjects/show
layout: layout
blueprints:
  - default
  - subject
revisions: false
route: '{parent_uri}/{slug}'
sort_dir: asc
structure:
  root: true
  tree:
    -
      entry: home
    -
      entry: 9ac8d33f-ff87-4a4f-b8b2-05b0c1b266e4
    -
      entry: 0f9f2a2a-b21a-4d99-9d9d-b95ffc75540c
    -
      entry: 3d9ad4cc-12c9-4d6d-99d6-4991ccefa1e4
```

**Comment by jasonvarga** [comment]
I think this issue is solved by 5a9d7a06. Feel free to reopen if not.
